src/main/java/uk/gov/justice/laa/crime/cfecrime/controllers/CfeCrimeController.java

```diff
@@ -32,9 +33,18 @@ public ResponseEntity<CfeCrimeResponse> invoke(@Parameter(
             content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
                     schema = @Schema(implementation = CfeCrimeRequest.class)
             )
-    ) @Valid @RequestBody CfeCrimeRequest request) {
-
-        CfeCrimeResponse response = RequestHandler.handleRequest(request);
-        return ResponseEntity.ok(response);
+    ) @RequestBody @Valid CfeCrimeRequest request, BindingResult bindingResult)  {
+        if (!bindingResult.hasErrors()) {
```

Comment:
don't understand this change. The request was already validated (using the pattern from CMA), so what does this achieve?

Reply:
The controller tests were failing for initial and full (as this was not set up correctly) and when I looked into it the request had errors so I handled this by letting the client know what the issue is so that they can fix. There was no indication that there was an error unless you look in the spring logs which only appears when running in debug mode.